packages/react/src/utilities/utility.module.css

```diff
@@ -1,24 +1,26 @@
-/**
+@layer fds.utility {
```

Comment:
```suggestion
@layer fds.utilities {
```

Reply:
Should we rename the file as well then?

Reply:
Better yet, just move it to a separate `utilities.css` in `packages/css`, but lets save that for a separate PR as it will touch a lot of files :)